src/collections/Breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,106 @@
+import _ from 'lodash'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+import META from 'src/utils/Meta'
+import { customPropTypes } from '../../utils/propUtils'
+import * as sui from '../../utils/semanticUtils'
+import BreadcrumbDivider from './BreadcrumbDivider'
+import BreadcrumbSection from './BreadcrumbSection'
+
+/**
+ * A breadcrumb is used to show hierarchy between content.
+ */
+function Breadcrumb(props) {
+  const {
+    children, divider, icon, size, sections, ...rest,
+  } = props
+
+  const classes = cx(
+    'ui',
+    size,
+    'breadcrumb'
+  )
+
+  if (!sections) {
+    return <div className={classes} {...rest}>{children}</div>
+  }
+
+  let dividerJSX = <Breadcrumb.Divider />
+
+  if (divider) {
+    dividerJSX = <Breadcrumb.Divider>{divider}</Breadcrumb.Divider>
+  }
+
+  if (icon) {
+    dividerJSX = <Breadcrumb.Divider icon={icon} />
+  }
```

Comment:
Looking at the `Breadcrumb.Divider` render method, it has conditional logic to return the proper component if an icon or children are passed.  It also has mutuallyExclusive prop types for these.

The `Breadcrumb` prop types mutually exclude `icon` and `divider`.  This means the only valid props are `icon` only, `divider` only or neither.  So we can simply do this:

```jsx
const dividerJSX = <Breadcrumb.Divider icon={icon}>{divider}</Breadcrumb.Divider>
```

Because, if nothing is passed, these will both be `undefined` and the default divider `/` will render.  Otherwise, only `icon` or `divider` can be passed in which case the other will be `undefined` and the component will still be valid.  There is no need for the if/else logic in this area.